packages/auth/lib/index.d.ts

```diff
@@ -1207,7 +1192,19 @@ export namespace FirebaseAuthTypes {
      * > It is recommended to use {@link auth#onAuthStateChanged} to track whether the user is currently signed in.
      */
     currentUser: User | null;
-
+    /**
+     * Sets the language code.
```

Comment:
What happens on native if `null` is passed? Does it use the app language?

Reply:
Android, errors when you pass `null` in: https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuth#public-void-setlanguagecode-string-languagecode

iOS doesn't error when you pass `null`. I can't actually find the documentation for the `FIRAuthRequestConfiguration` class that holds the language code value so I assume it is fairly old, I did find it in the iOS source code: https://github.com/firebase/firebase-ios-sdk/blob/42655733d51214ee83362afab0b0f545ce3b2102/FirebaseAuth/Sources/Backend/FIRAuthRequestConfiguration.h#L36 which confirms it is `nullable`. I've updated the code to handle both.